tracer/test/test-applications/instrumentation/CallTargetNativeTest/With0Arguments.cs

```diff
@@ -39,4 +39,45 @@ class With0ArgumentsThrowOnAsyncEnd
     {
         public Task Wait2Seconds() => Task.Delay(2000);
     }
+
+    partial class ArgumentsParentType
```

Comment:
This may blow up the number of test cases, but could you also do a nested type where the parent type is a struct?

Reply:
done in [ad13c83](https://github.com/DataDog/dd-trace-dotnet/pull/2818/commits/ad13c834c483658229db059e5137cc52d565e0a0)